### DISCUSSION TITLE
Is there a way do use it for a Dual Pc Setup?

### DISCUSSION THREAD

**Discussion by MadeByLukeDEV** [tryptech/obs-zoom-and-follow#49] [Q&A] [answered]
I record/Stream on a Dual Setup. the problem is the Elgato is not includes in OBS as a Display Capture, it is a Capture Device.

So i tryed to write in the Source for Display: my elgato Source. fine it worked he zoom in and out BUT! not right.... he zooms from 1920x1080 too 1x1 and after that he zoom crop my image fixed to 1280x720 :/

I also tryed once with a Display Capture but the same problem

Python, Screeninfo and pywinctl are installed.

here are some screenshots:
settings: https://ibb.co/rHSH63Z
source settings: https://ibb.co/qm4QT3Z

**Comment by tryptech** [comment, marked as answer]
No way to do it via 2 PCs for now. There is no way implemented right now to transfer mouse position information from one computer to another right now.
